pages/who-we-are/organisation-structure.md

```diff
@@ -2,108 +2,19 @@
 title: DTA organisation structure
 layout: base
 permalink: /who-we-are/organisation-structure/
-searchexcerpt: "Please find an overview of high-level organisation structure of DTA."
+searchexcerpt: "Find an overview of high-level organisation structure of DTA. The DTA consists of 7 key areas covering finance, people, communication, strategy, digital products and skills, investment and procurement."
 ---
-<article id="content" class="content-listing home">
-<header class="about-dta">
-<h1>DTA organisation structure</h1>
-</header>
 
-<div class="org-chart">
-  <h2 class="home-heading">Chief Executive Officer - Gavin Slater</h2>
-    <div class="org-counter">
+<article id="content" class="content-listing home" markdown="1" >
 
-      <div class="org-column">
 
-        <h3 class="org">Enabling Services<br><br></h3>
+## DTA organisation structure
 
-        <h4>Radi Kovacevic</h4>
-        <br><br><br><br>
-        <hr style="border-top: 1px #fff solid; border-bottom:6px #5bcbe3 solid;">
-        <ul><li>Finance, risk and procurement</li>
-          <li>People Services</li>
-          <li>Business Services and Technology</li>
-          <li>Ministerial and parliamentary services</li>
-        </ul>
-        <hr style="border-top: 1px #fff solid; border-bottom:3px #5bcbe3 solid;">
-      </div>
 
+![Image showing the DTA organisation structure as on 1 July 2017](/images/DTA_Orgchart_Julyv2017.svg)
```

Comment:
And this alt-text does not completely describe what's in the image. For screenreaders we need to have a more complete text description in the image, e.g. similar to https://www.dta.gov.au/what-we-do/transformation-agenda/#government-digital-transformation-roadmap. It could be an accordion or a link off to another page.